damnit/ctxsupport/ctxrunner.py

```diff
@@ -465,6 +465,27 @@ def generate_thumbnail(image):
     return figure2png(fig, dpi=THUMBNAIL_SIZE)
 
 
+def line_thumbnail(arr):
+    from matplotlib.figure import Figure
+
+    # width = 3 * height; roughly fits table cells
+    fig = Figure(figsize=(2, 2/3))
+    ax = fig.add_subplot()
+    fig.subplots_adjust(left=0, right=1, bottom=0, top=1)
+
+    if isinstance(arr, np.ndarray):
+        ax.plot(arr)
+    else:
+        # Use DataArray's own plotting method
+        arr.plot(ax=ax)
```

Comment:
Is this necessary? I think for 1D plots the only extra thing xarray does is add the right axis labels and ticks, but those are disabled in the preview.

Reply:
I think it will use the coordinate labels as the x points, so if they're not evenly spaced - e.g. if you've done a scan with varying step sizes - this could make a difference. Probably not common, but I don't think it does any harm.